src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -242,6 +242,37 @@ public function dontSeeJson(array $data = null)
     }
 
     /**
+     * Assert that the json response has a certain structure.
+     *
+     * @param array|null $structure
```

Comment:
StyleCI is happy because I've not written code that checks all our phpdoc rules yet. :P